service/api_task.go

```diff
@@ -202,21 +186,13 @@ func (as *APIServer) EndTask(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// mark task as finished
-	startPhaseAt = time.Now()
```

Comment:
Should we keep logging around the entirety of `MarkEnd`?

Reply:
There's still logging around the entirety of task.MarkEnd, which was the slow part